app/helpers/script_helper.rb

```diff
@@ -31,14 +31,18 @@ def render_javascript_pack_once_tags(*names)
 
   private
 
+  SAME_ORIGIN_ASSETS = %w[
+    identity-style-guide/dist/assets/img/sprite.svg
+  ]
+
   def local_crossorigin_sources?
     Rails.env.development? && ENV['WEBPACK_PORT'].present?
   end
 
   def javascript_assets_tag(*names)
     assets = AssetSources.get_assets(*names)
     if assets.present?
-      asset_map = assets.index_with { |path| asset_path(path) }
+      asset_map = assets.index_with { |path| asset_path(path, host: asset_host(path)) }
```

Comment:
For extra context on why we have to do this, see #5895 and related discussions / resources.